great_expectations/core/batch_spec.py

```diff
@@ -107,3 +107,21 @@ def query(self):
     @query.setter
     def query(self, query) -> None:
         self["query"] = query
+
+
+class GlueCatalogBatchSpec(BatchSpec, metaclass=ABCMeta):
```

Comment:
@lccasagrande I see -- great that we are now making clear that this is "Glue Catalog" integration.  Thank you for that.

Reply:
@lccasagrande Quick question: Why do we need `GlueCatalogBatchSpec` to be a a `metaclass`?  I am not seeing any methods declared as `abstract`, nor do I see any prior initialization (`new`) code for the descendants.  Thank you.

Reply:
Fixed